guide/english/machine-learning/pytorch-for-deep-learning/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Deep Learning using PyTorch
+---
+
+
+#### What is PyTorch?
+"PyTorch is open source deep learning platform primarily developed by Facebook's artificial-intelligence research group,that provides a seamless path from research prototyping to production deployment which has been adopted by many (if not most) of the researchers that we most respect."
+
+
+* In PyTorch things are way more imperative and dynamic: you can define, change and execute nodes as you go, no special session interfaces or placeholders like in Tensorflow. Overall, the framework is more tightly integrated with Python language and feels more native most of the times.

```

Comment:
For "In PyTorch **things** are way more imperative.....", perhaps you can briefly mention what "things" mean. Do you mean the process or something else? Otherwise it might be confusing for the user